tests/visco_plastic_differentplasticitydifferentphases.prm

```diff
@@ -0,0 +1,105 @@
+# Copy of visco_plastic_phases.prm, but tests different plasticity parameters for different phases.
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 10
+set Output directory                       = visco_plastic
```

Comment:
Set to the name of the test? (although this does not matter when the tester is run, parameter is ignored)